compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/Scrollbar.desktop.kt

```diff
@@ -174,9 +174,10 @@ fun VerticalScrollbar(
  * @param interactionSource [MutableInteractionSource] that will be used to dispatch
  * [DragInteraction.Start] when this Scrollbar is being dragged.
  */
+@Deprecated("Use androidx.compose.foundation.v2.HorizontalScrollbar instead")
```

Comment:
Please, replace deprecated API in our internal example (example1/Main.jvm.kt)

Reply:
Will do.

Reply:
No need, after the changes to `VerticalScrollbar` and `rememberScrollbarAdapter`.